build/check-fmt.sh

```diff
@@ -8,7 +8,7 @@ function opa::check_fmt() {
     exit_code=0
     for pkg in $(opa::go_packages); do
         for file in $(opa::go_files_in_package $pkg); do
-            __diff=$(gofmt -d $file | tee >(cat - >&5))
+            __diff=$(goimports -d $file | tee >(cat - >&5))
```

Comment:
Does this need `-local github.com/open-policy-agent/opa`, or do I remember something that is no longer up-to-date? From my local `goimports --help`:
```
  -local string
        put imports beginning with this string after 3rd-party packages; comma-separated list
```

(Apparently, it's not breaking stuff right now, but I believe that without the `-local` flag, this check might be too weak?)

Reply:
Agreed, I think we might want to have the check (and changes for this PR to fix existing issues) be from something like `goimports -local github.com/open-policy-agent/opa -format-only -d ..`